src/debug/ee/controller.cpp

```diff
@@ -4486,7 +4488,7 @@ void DebuggerPatchSkip::DebuggerDetachClean()
    // THIS FIX IS INCOMPLETE!It attempts to update the IP in the cases we can easily detect.However, 
    // if a thread is in pre - emptive mode, and its filter context has been propagated to a VEH 
    // context, then the filter context we get will be NULL and this fix will not work.Our belief is 
-   // that this scenario is rare enough that it doesn�t justify the cost and risk associated with a 
+   // that this scenario is rare enough that it doesn�t justify the cost and risk associated with a 
```

Comment:
It seems like you have not used the plain apostrophe character here, could you please fix that?